packages/meditrak-server/src/routes/exportSurveys/RowBuilder.js

```diff
@@ -53,7 +53,6 @@ export class RowBuilder {
     const processedOptions = options.map((option, optionIndex) => {
       try {
         const { value, label, color } = JSON.parse(option);
-        if (value === undefined) return option; // May be JSON but still not a formatted option object
```

Comment:
Is this case no longer possible?

Reply:
Just this question pending @chris-bes

Reply:
Ah sorry, I was going to make comments once I've fixed up all the reviews. Yep I don't think this is possible anymore. Had a search in the `question` database and there's no cases that an `option` is in JSON format but has no `value` property. So I think it's safe to remove it

Reply:
Great! Sorry, I jumped on this before you requested re-review - bad form.